tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/model/LiferayPortalSetup.java

```diff
@@ -0,0 +1,67 @@
+package com.liferay.ide.project.core.model;
+
+/**
+ * @author Lovett Li
+ */
+public class LiferayPortalSetup
+{
+
+    private String previousName;
```

Comment:
Lets create a new sub-object that is specific to the fields needed for these 4. something like "PortalSettings"